modelci/hub/converter/to_trt.py

```diff
@@ -32,11 +34,42 @@
 )
 from modelci.utils import Logger
 
+
 logger = Logger('converter', welcome=False)
 
 
 class TRTConverter(object):
-    supported_framework = ["onnx", "tfs"]
+    supported_framework = ["onnx", "tfs", "tensorflow"]
+
+
+    @staticmethod
+    def from_tensorflow(
+            savedmodel_path: Path,
+            shape: List,
+            opset: int = 10,
+    ):
+        """
+        This is the function to create the TensorRT engine
+        Args:
+           savedmodel_path : Path to savedmodel_file.
+           shape : Shape of the input of the savedmodel file.
+       """
+        import tensorrt as trt
+
+        tmpdir = tempfile.mkdtemp()
+
+        onnx_save = str(os.path.join(tmpdir, "temp_from_tf/")) + '/tempmodel.onnx'
+        convertcmd = ['python', '-m', 'tf2onnx.convert', '--saved-model', savedmodel_path, '--output', onnx_save,
+                      '--opset', str(opset)]
+        subprocess.run(convertcmd)
```

Comment:
Do they provide Python SDK? Python API is more favor than use subprocess call.

Reply:
Yes, they just provide Python SDK  29 days ago on github version, yet the release version on pypi do not support this. I recommend to wait for the stable version before we use the SDK.

Reply:
may add a TODO for further inspection.